src/template.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html> 
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Hello world!</title>
+  </head>
+  <body>
+    <h1>Leaderboard</h1>
+
+  <section class="board">
+
+    <div class="container">
+      <div class="recent-score">
+        <h2>Recent scores</h2>
+        <button>Refresh</button>
+      </div>
+
+      <div class="scores">
+        <table>
+          <tr><td>Name: 100</td></tr>
+          <tr><td>Name: 20</td></tr>
+          <tr><td>Name: 50</td></tr>
+          <tr><td>Name: 78</td></tr>
+          <tr><td>Name: 125</td></tr>
+          <tr><td>Name: 77</td></tr>
+          <tr><td>Name: 42</td></tr>
```

Comment:
- Hey, although this wasn't mentioned in the project requirement, do have in mind that this section should be populated dynamically in the JS file. please, kindly update this.

Reply:
Hi @zieeco, I think this should be done tomorrow not today, in the requirement we shouldn't work with API today.

Reply:
@Hisoka37, please, kindly create an array of scores, then populate and use the data to populate your scores. this should not be hardcoded as you've done in the template.html file as you've done. Remember, this is a javascript project.

Reply:
@zieeco yeah you're right it's a javascript project, thank you ❤️

Reply:
@Hisoka37, I hope this helped a lot! 

Cheers and Happy Coding!